docs/sharding.md

```diff
@@ -0,0 +1,42 @@
+# Multiple Azure SignalR Service Instances Support
+Currently we support multiple Azure SignalR Service instances **for Persistent mode only**.
```

Comment:
Add a section about limitations? Beside mode, `Java` is not supported in the first phase.

Reply:
Java or other languages are supported. They just can't customize router.